src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -632,6 +644,36 @@ export function getEventContent(event) {
         title: 'Your Motion for Reconsideration was denied',
         description: '',
       };
+    case EVENT_TYPES.rampNotice:
+      return {
+        title: 'VA sent you a letter about the Rapid Appeals Modernization Program',
+        description: '',
+      };
+    case EVENT_TYPES.rampOptIn: // TODO: should this simply be "ramp"?
+      return {
+        title: 'You opted in to the Rapid Appeals Modernization Program',
+        description: '',
+      };
+    case EVENT_TYPES.otherClose:
+      return {
+        title: 'Your appeal was closed',
+        description: '',
+      };
+    case EVENT_TYPES.vacated:
+      return {
+        title: 'Board of Veterans’ Appeals vacated a previous decision',
```

Comment:
Is there any other way to say this? How's a Vet going to know what "vacated a previous decision" means?

Reply:
@gnakm Does it make sense to write another ticket to circle back around to this later, or do we not expect to revise this?